### PR TITLE
Flatpak build and run glue

### PR DESCRIPTION
**Pull request by mkhon**

### CODE REVIEW

**Comment by soredake** [comment]
Any progress on this?

**Comment by az0** [comment]
I believe Max did a good job, but I haven't had a chance to build and test this yet.

File: org.bleachbit.BleachBit.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "app-id": "org.bleachbit.BleachBit",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```
Comment (LeandroStanger) [inline org.bleachbit.BleachBit.json:4]: Update runtime to 20.08
```suggestion
  "runtime-version": "20.08",
```

**Comment by az0** [comment]
Completed https://github.com/bleachbit/bleachbit/issues/632

**Pull request closed without merging**